packages/fxa-payments-server/src/index.scss

```diff
@@ -49,144 +49,142 @@ h3.billing-title {
     flex-direction: row;
     padding-top: $header-height-tablet;
   }
+}
 
-  hr {
-    border: 0;
-    border-top: 1px solid #e0e0e6;
-    height: 0;
-    margin: 0 -40px;
+hr {
+  border: 0;
+  border-top: 1px solid #e0e0e6;
+  height: 0;
+  margin: 0 -40px;
 
-    @include respond-to('small') {
-      margin: 0 -20px; // match padding of .main-content
-    }
+  @include respond-to('small') {
+    margin: 0 -20px; // match padding of .main-content
   }
+}
 
-  .input-row {
-    // Stripe element class names
-    // https://stripe.com/docs/stripe-js/reference#the-element-container
-    margin: 24px 0;
+.input-row {
+  // Stripe element class names
+  // https://stripe.com/docs/stripe-js/reference#the-element-container
+  margin: 24px 0;
 
-    .StripeElement {
-      @include input-element();
-      padding-top: 14px;
+  .StripeElement {
```

Comment:
I wonder if this is an overqualified selector - does this need to exist inside `.input-row`?